app/src/main/java/org/simple/clinic/bloodsugar/history/adapter/BloodSugarHistoryListItem.kt

```diff
@@ -65,12 +65,16 @@ sealed class BloodSugarHistoryListItem : ItemAdapter.Item<Event> {
         bpEditableFor: Duration,
         utcClock: UtcClock
     ): Boolean {
-      val now = Instant.now(utcClock)
-      val createdAt = bloodSugarMeasurement.timestamps.createdAt
+      return if (bloodSugarMeasurement.reading.type !is Unknown) {
```

Comment:
Negation statements add some cognitive complexity so we should avoid them if possible.
Do you mind reversing this condition?
```
if (bloodSugarMeasurement.reading.type is Unknown) { 
   false 
} else {
...
} 
```